src/Campaigns/resources/admin/components/CampaignDetailsPage/Components/DefaultForm.tsx

```diff
@@ -1,12 +1,25 @@
 import {__} from "@wordpress/i18n";
 import HeaderText from './HeaderText';
 import HeaderSubText from './HeaderSubText';
-import {CampaignFormOption} from "@givewp/campaigns/admin/components/CampaignDetailsPage/types";
+import {GiveCampaignDetails} from "@givewp/campaigns/admin/components/CampaignDetailsPage/types";
+import {useFormContext} from "react-hook-form";
+
+declare const window: {
+    GiveCampaignDetails: GiveCampaignDetails;
+} & Window;
 
 /**
  * @unreleased
  */
-const DefaultFormWidget = ({defaultForm}: {defaultForm: string}) => {
+const DefaultFormWidget = () => {
+
+    const {watch} = useFormContext();
+
+    const [defaultFormId] = watch(['defaultFormId']);
```

Comment:
Any reason to not use the `useWatch` hook instead? I typically use the hook for the performance benefits unless I need to watch many values at the root level:

From the docs about [watch](https://react-hook-form.com/docs/useform/watch)
>This API will trigger re-render at the root of your app or form, consider using a callback or the [useWatch](https://react-hook-form.com/docs/usewatch) api if you are experiencing performance issues.

Reply:
That would probably work. I had copy/pasted from another part of the codebase, but I'll refactor and try it.